src/modules/vision_target_estimator/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+############################################################################
+#
+#   Copyright (c) 2018 PX4 Development Team. All rights reserved.
```

Comment:
2023

Reply:
Updated in https://github.com/PX4/PX4-Autopilot/pull/22008/commits/943ae46fc69aded45be7a12308151b30e5ca21f6